### PR TITLE
Add 3 posts and remove 1 event

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: _posts/2020-07-29-galaxy-20-05.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-site: freibu[pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]rg
+site: freibu[pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
```
Comment (bgruening) [inline _posts/2020-07-29-galaxy-20-05.md:2]: freibu - should also be removed.

File: _events/2020-11-05-devs-roundtable.md [hunk 2]
```diff
@@ -1,12 +0,0 @@
----
-site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
-tags:
-title: "Galaxy Developer Roundtable: Developer Training"
```
Comment (bgruening) [inline _events/2020-11-05-devs-roundtable.md:4]: why removing that?

Comment (beatrizserrano) [reply _events/2020-11-05-devs-roundtable.md:4]: fixed

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Merci!

**Pull request merged**